backend/game/scribble/handler/gamemaster/gamemaster.go

```diff
@@ -335,23 +339,59 @@ func (h *Handler) Name() string {
 	return name
 }
 
+type WaitForStartSend struct {
+	ClientUUID string `json:"clientUUID"`
+	IsReady    bool   `json:"isReady"`
+}
+
 type WaitForStartReceive struct {
-	StartGame bool `json:"startGame"`
+	clientUUID string
+	IsReady    bool `json:"isReady"`
 }
 
 // waitForStart will wait for the leader to press the start button.
 // when the leader has pressed the start button, there should be a
 // incoming message on the h.waitForStartChan in which we can
 // continue onto the next gamestate
 func (h *Handler) waitForStart() {
-	select {
-	case msg := <-h.waitForStartChan:
-		if msg.StartGame == true {
+	for {
+		select {
+		case msg := <-h.waitForStartChan:
+			for idx, client := range h.clientList.clients {
+				if client.UUID == msg.clientUUID {
+					h.clientList.clients[idx].isReady = msg.IsReady
+					playerReadyStateChangedMsg := Send{
+						GameMasterAPI: WaitForStart,
+						WaitForStartSend: WaitForStartSend{
+							ClientUUID: msg.clientUUID,
+							IsReady:    msg.IsReady,
+						},
+					}
+					playerReadyStateChangedBytes, err := game.MessageBuild("game", playerReadyStateChangedMsg)
```

Comment:
this line is too long. 
![image](https://user-images.githubusercontent.com/12277640/89108241-fc33fa80-d3f3-11ea-9bea-852ba9433f02.png)

can we do something like this:

```go
					playerReadyStateChangedBytes, err :=
						game.MessageBuild("game", playerReadyStateChangedMsg)
					if err != nil {
						log.Fatalf("unable to marshal: %v", err)
						return
					}
					h.reg.SendToSameHub(
						client.ClientUUIDStruct,
						playerReadyStateChangedBytes,
					)
```